wrapper/src/test/java/software/amazon/jdbc/plugin/IamAuthConnectionPluginTest.java

```diff
@@ -154,6 +157,17 @@ public void testConnectWithSpecifiedHost() throws SQLException {
         "pg.testdb.us-east-2.rds.amazonaws.com");
   }
 
+  @Test
+  public void testAwsSupportedRegionsUrlExists() throws IOException {
+    URL url =
+        new URL("https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html");
+    HttpURLConnection urlConnection = (HttpURLConnection) url.openConnection();
+    urlConnection.setRequestMethod("HEAD");
```

Comment:
Is this required?

Reply:
it improves response time, but for just one get request it isn't needed. removed